<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastFmView</class>
 <widget class="QWidget" name="LastFmView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="MySplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>User:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>header</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Loved tracks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weekly chart</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Recent tracks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tagged tracks</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
          <item>
           <property name="text">
            <string>neurofunk</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Playlists</string>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
          <item>
           <property name="text">
            <string>best of</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="CustomTableView" name="tableView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTableView</class>
   <extends>QTableView</extends>
   <header>customtableview.h</header>
  </customwidget>
  <customwidget>
   <class>MySplitter</class>
   <extends>QSplitter</extends>
   <header>mysplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
